contrib/mpi-proxy-split/mpi-wrappers/mpi_request_wrappers.cpp

```diff
@@ -210,6 +211,7 @@ USER_DEFINED_WRAPPER(int, Wait, (MPI_Request*) request, (MPI_Status*) status)
       fflush(stdout);
 #endif
     }
+    LOG_POST_Wait(request, statusPtr);
```

Comment:
And similarly to above, we don't want to use log-replay here for MPI_Wait.  Please remove this.

Reply:
Currently only LOG_POST_Wait() takes the request status->{tag, source}. It is a good idea to remove this line. However, if we remove this line, then we have to change LOG_POST_Irecv to retrieve the {tag, source} from status and save it in the message log. If we go through this way, we do not need p2p_log_request_<rank>.txt created by save_request_info() function which is called by LOG_POST_Wait(). Further we could remove mana_p2p_update_log.c alltogether.